js/views/reader_view.js

```diff
@@ -280,6 +280,8 @@ var ReaderView = function (options) {
             _.defer(function () {
                 Globals.logEvent("PAGINATION_CHANGED", "EMIT", "reader_view.js");
                 self.emit(Globals.Events.PAGINATION_CHANGED, pageChangeData);
+                
+                if (!pageChangeData.spineItem) return;
                 _.defer(function () {
```

Comment:
`pageChangeData.spineItem` is also used in the Media Overlays player, via the line above (`_mediaOverlayPlayer.onPageChanged(pageChangeData);`). Thankfully, there are safeguards inside the MO code, so no crashes. Unfortunately, we use `var spineItems = reader.getLoadedSpineItems();` to match `pageChangeData.spineItem` in multiple page spreads (fixed layout), so it would be nice if the `spineItem` property was set correctly in all view types (inc. reflow and scroll).

Reply:
Note that I fixed the reflow view: https://github.com/readium/readium-shared-js/commit/6d8beaa486716681ed696a5408f97e4637e64803